src/coredump/coredump.c

```diff
@@ -1154,6 +1216,12 @@ static int send_iovec(const struct iovec_wrapper *iovw, int input_fd) {
         if (r < 0)
                 return log_error_errno(r, "Failed to send coredump fd: %m");
 
+        if (mounts_fd >= 0) {
+                r = send_many_fds(fd, (int[]) { input_fd, mounts_fd }, 2, 0);
```

Comment:
Yes and I did this because I wanted to make sure the "protocol" is backward compatible. I was thinking about crashes that happen during package updates/downgrades when potentially systemd-coredump forked off by the kernel is different than socket activated systemd-coredump.